index.js

```diff
@@ -3,6 +3,26 @@
 var clone = require('101/clone')
 var debug = require('debug')('swarmerode')
 var exists = require('101/exists')
+var Promise = require('bluebird')
+var cache = {}
+
+function handleCache (key, cacheFetch, cb) {
```

Comment:
document this function. I'd actually consider moving it to it's own file as well, which would make it easy to unit test